Assets/MRTK/Examples/Demos/UX/Slider/Scripts/SliderChangePalmThresholdValue.cs

```diff
@@ -0,0 +1,21 @@
+﻿//
```

Comment:
Is this related to your main change? Was this accidentally left in during debugging / tuning, or is it intentional?

Reply:
Not sure what happened here; I think this was a byproduct of it going to mrtk_development rather than the march hand menu branch

Reply:
Sorry, this is my temporary change to use the slider to test out hand drop (camera facing threshold) value. I'll remove before opening PR.